dclab/rtdc_dataset/core.py

```diff
@@ -748,7 +653,7 @@ def get_kde_scatter(self, xax="area_um", yax="deform", positions=None,
         positions: list of two 1d ndarrays or ndarray of shape (2, N)
             The positions where the KDE will be computed. Note that
             the KDE estimate is computed from the points that
-            are set in `self.filter.all`.
```

Comment:
please revert this change, it slipped back in when you copy-pasted the doc string